tokio/src/sync/mpsc/bounded.rs

```diff
@@ -711,6 +711,16 @@ impl<T> Receiver<T> {
     ) -> Poll<usize> {
         self.chan.recv_many(cx, buffer, limit)
     }
+
+    /// Returns the number of [`Sender`] handles.
+    pub fn strong_count(&self) -> usize {
```

Comment:
Wouldn't it be more explicit to name it `sender_strong_count` here?

Reply:
I agree that `sender_strong_count` and `sender_weak_count` would be better. Otherwise, this PR looks good to me.

Reply:
Sounds good. I will update it. Thank you!

Reply:
Updated.